test/parallel_api/ranges/std_ranges_copy.pass.cpp

```diff
@@ -0,0 +1,44 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "std_ranges_test.h"
+
+std::int32_t
+main()
+{
+#if _ENABLE_STD_RANGES_TESTING
+    using namespace test_std_ranges;
+    namespace dpl_ranges = oneapi::dpl::ranges;
+
+    auto copy_checker = [](std::ranges::random_access_range auto&& __r_in,
+                           std::ranges::random_access_range auto&& __r_out, auto&&... args)
+    {
+        const auto _size = std::ranges::min(std::ranges::size(__r_in), std::ranges::size(__r_out));
```

Comment:
Minor stylistic suggestion: there is no need to uglify the tests, let's avoid it.

Reply:
yes, it makes sense